cmd/podman/system/service_abi.go

```diff
@@ -106,6 +108,13 @@ func restService(flags *pflag.FlagSet, cfg *entities.PodmanConfig, opts entities
 	// Close the fd right away to not leak it during the entire time of the service.
 	devNullfile.Close()
 
+	cgroupv2, _ := cgroups.IsCgroup2UnifiedMode()
+	fmt.Printf("rootless.IsRootless() && !cgroupv2 %v %v\n", rootless.IsRootless(), cgroupv2)
```

Comment:
debug output?

Reply:
yes sorry for leaving that, dropped now.